comp/core/gui/guiimpl/views/private/js/javascript.js

```diff
@@ -19,10 +19,51 @@ function sendMessage(endpoint, data, method, callback, callbackErr){
     type: method,
     data: data,
     success: callback,
-    error: callbackErr
+    error: function(requestObject, error, errorThrown) {
+      try {
+        callbackErr(requestObject, error, errorThrown)
+      } catch (error) {
+        console.error(error)
+      }
+      $("#agent_status").html("Not connected<br> to Agent");
+      $("#agent_status").css({
+        "background": 'linear-gradient(to bottom, #c62d1f 5%, #f24437 100%)',
+        "background-color": '#c62d1f',
+        "border": '1px solid #d02718',
+        "text-shadow": '0px 1px 0px #810e05',
+        'left': '-180px'
+      })
+      // erase right side of the page
+      $(".page").hide();
+      $(".right").html("");
+      setError(requestObject.status, requestObject.responseText)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** Why do we need that ?‏

Reply:
It's how the GUI work (old fashion isn't it ^^'). Every time the user click on a menu button (let's say `logs`) the JS script will erase the right part of the layout and write in it.

Reply:
> It's how the GUI work (old fashion isn't it ^^'). Every time the user click on a menu button (let's say logs) the JS script will erase the right part of the layout and write in it.

Old school React 😄